apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -1391,31 +1398,56 @@ describe('GameLabSprite', function () {
       }).to.throw(TypeError, 'Usage: setCollider("circle") or setCollider("circle", offsetX, offsetY, radius)');
     });
 
+    it('can construct a rectangle collider with default dimensions and offset', function () {
+      sprite.setCollider('rectangle');
+      expect(sprite.collider).to.be.an.instanceOf(gameLabP5.p5.AABB);
+      expect(sprite.collider.center).to.eq(sprite.position);
+      // Extents should be sprite dimensions
+      expect(sprite.collider.extents).to.be.an.instanceOf(p5.Vector);
+      expect(sprite.collider.extents.x).to.eq(sprite.width);
+      expect(sprite.collider.extents.y).to.eq(sprite.height);
+      // Offset should be zero
+      expect(sprite.collider.offset).to.be.an.instanceOf(p5.Vector);
+      expect(sprite.collider.offset.x).to.eq(0);
+      expect(sprite.collider.offset.y).to.eq(0);
+    });
+
+    it('scaling sprite without animation does not affect default rectangle collider size', function () {
```

Comment:
Again, the collider's internal size representation reflects the sprite's unscaled size.  Here's a version of your example using default colliders (whose behavior we're trying to emulate here): https://studio.code.org/projects/gamelab/xN4-oN_NwTJTwTY5Ld4SDQ/view